docs/migration/5_0.md

```diff
@@ -813,3 +813,10 @@ Due to i18next migration, certain set of keys have been migrated (from `_plural`
 - Added `RegisterComponentService` that accepts `UserRegisterFacade` through constructor injection. `RegisterComponentService` has only those methods from `UserRegisterFacade` that are needed for `RegisterComponent`.
 - Updated the constructor of `RegisterComponent` and replaced `UserRegisterFacade` with `RegisterComponentService`.
 
+## Breaking changes in cdc intrgration library
```

Comment:
Please also add the migrations, like you did here: https://github.com/SAP/spartacus/pull/15785/files#diff-77459068ca72c30a6a06cb46550b5357cada208b72fbde51b56252bbdbb3d371